docs/docs/interactive/applications.md

```diff
@@ -143,21 +143,21 @@ app = Application(
 ```
 #### Best practices
 
-Regardless of user instructions, the AI application is told that it is the natural-language interface to an application, rather than the application itself. This tends to increase compliance and help it interpret user intent. Moreover, it means that user instructions can freely acknowledge the LLM's role as an interface or describe the application in an LLM-independent manner.
+Regardless of user instructions, the AI application is told that it is the natural-language interface to an application, rather than the application istelf. This tends to increase compliance and help it interpret user intent. Moreover, it means that user instructions can freely acknowledge the LLM's role as an interface or describe the application in an LLM-independent manner.
```

Comment:
i think you had correctly identified a misspelling here the first time

Reply:
all of them are now resolved sorry for missing them out